target/linux/pistachio/modules.mk

```diff
@@ -175,3 +175,49 @@ endef
 
 $(eval $(call KernelPackage,nhc_udp))
 
+define KernelPackage/uccp420wlan
+  SUBMENU:=$(WIRELESS_MENU)
+  TITLE:=Imagination wireless driver
+  KCONFIG:=CONFIG_UCCP420WLAN
+  KCONFIG:=\
+    CONFIG_UCCP420WLAN=y \
```

Comment:
Should this be a module? anything that is to be built as a module can be left blank i.e. "CONFIG_UCCP420WLAN \"

OpenWRT will fill out modules automatically (=m).